.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Eslint
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        node-version: [22]
+    steps:
+    - uses: actions/checkout@v4
+    - name: Install pnpm
```

Comment:
I would have expected that you first need to install Node.js and then pnpm, but it seems to work that way, too. So it's probably fine.